projects/plugins/jetpack/modules/comments/comments.php

```diff
@@ -277,11 +277,11 @@ public function comment_form_after() {
 			$params['has_cookie_consent']  = (int) ! empty( $commenter['comment_author_email'] );
 		}
 
-		$blog_token = Jetpack_Data::get_access_token();
+		$blog_token        = ( new Tokens() )->get_access_token();
 		list( $token_key ) = explode( '.', $blog_token->secret, 2 );
 		// Prophylactic check: anything else should never happen.
 		if ( $token_key && $token_key !== $blog_token->secret ) {
-			// Is the token a Special Token (@see class.jetpack-data.php)?
+			// Is the token a Special Token (@see class.tokens.php)?
```

Comment:
Let's add a mental note to answer this question in a public method in `Tokens` in the future :)